pkg/storaged/client.js

```diff
@@ -1765,10 +1765,22 @@ client.should_ignore_block = (block) => {
 
 client.export_mount_point_mapping = () => {
     const mpm = { };
+
     for (const p in client.blocks) {
         const b = client.blocks[p];
+
         for (const c of b.Configuration) {
-            if (c[0] == "fstab") {
+            if (c[0] == "crypttab") {
+                const cryptoDevicePath = Object.keys(client.blocks_crypto).find(cb => client.blocks_crypto[cb].path === b.path);
+                const cryptoDeviceObject = client.blocks_crypto[cryptoDevicePath];
+                const deviceObject = client.blocks[cryptoDevicePath];
+                const device = utils.decode_filename(b.PreferredDevice);
+                const dirEncrypted = cryptoDeviceObject.ChildConfiguration?.[0]?.[1].dir.v;
+                const dir = dirEncrypted && client.strip_mount_point_prefix(utils.decode_filename(dirEncrypted));
+                const type = deviceObject.IdType;
+
+                mpm[device] = { dir, type, passphrase: "fillme" };
```

Comment:
@mvollmer as a follow up to our discussion today - I sent this WIP work which handles devices in crypttab.

I probably made it much more complicated that it needs to be, can you help? 

I need the luks device (/dev/mapper/luks-xyz) (now this get's the parent partition path), the mount point and the passphrase. Is that possible?